tests/data_platform/generate_features/test_copy_flat_features.py

```
@@ -0,0 +1,127 @@
+        features_root,
+        "is_likely_spam",
+        [{"uri": URI_POST_A, "label_timestamp": LABEL_TIMESTAMP, "is_likely_spam": False}],
+    )
+    run_dir = features_root / "2026_02_01-00:00:00"
+    write_feature_csv(
+        run_dir,
+        "is_political",
+        [{"uri": URI_POST_A, "label_timestamp": LABEL_TIMESTAMP, "is_political": False}],
+    )
+
+    with pytest.raises(FileExistsError, match="is_political.csv"):
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)